### PR TITLE
.osx: Show the /Volumes folder

### PR DESCRIPTION
**Pull request by nowherecat**
e.g. to add it to the Sidebar

### CODE REVIEW

File: .osx [hunk 1]
```diff
@@ -326,6 +326,9 @@ sudo nvram boot-args="mbasd=1"
 # Show the ~/Library folder
 chflags nohidden ~/Library
 
+# Show the /Volumes folder
+chflags nohidden /Volumes
```
Comment (mathiasbynens) [inline .osx]: Surely this needs a `sudo`?

**Comment by nowherecat** [comment]
Thanks for pointing it out. I wrongly assumed that asking for the administrator password in the very beginning is enough

File: .osx (hunk 1 above)
Comment (nowherecat) [inline .osx]: Thanks for pointing it out. I wrongly assumed that asking for the administrator password in the very beginning is enough

**Comment by mathiasbynens** [comment]
:+1:

**Pull request closed without merging**